app/experimenter/static/js/tests/Error.test.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import {render} from "@testing-library/react";
+import '@testing-library/jest-dom/extend-expect';
+import Error from "experimenter/components/Error";
+
+
+describe("<Errors>",()=>{
```

Comment:
this is generally the right approach, just a couple little tweaks:
```js
describe("The `Error` component", () => {
    const error = "an error occured!";
    it("renders the error message",()=>{
        const { getByText } = render(<Error error={error} />);
        expect(getByText(error)).toBeInTheDocument();
    });
})
```

Just a small note on the changes:
- `"<Errors>"` doesn't really make it clear what is being tested since `Errors` is not an actual component. Also on that note typically the string for the `it` method should be something that completes the sentence from the phrase in `describe`.
- I like to be explicit about the props that are being passed to the component so I went with `<Error error={error} />` instead of using a spread operator expand error props.
- I re-used the `error` variable in the `expect` method to keep things more DRY.